WcaOnRails/spec/models/merge_people_spec.rb

```diff
@@ -25,14 +25,14 @@
     person = FactoryGirl.create(:person_with_multiple_sub_ids)
     merge_people.person1_wca_id = person.wca_id
     expect(merge_people).to be_invalid
-    expect(merge_people.errors.messages[:person1_wca_id]).to eq ["This person has multiple subIds"]
+    expect(merge_people.errors.messages[:person1_wca_id]).to include "This person has multiple subIds"
   end
 
   it "requires person2 not have multiple subIds" do
     person = FactoryGirl.create(:person_with_multiple_sub_ids)
     merge_people.person2_wca_id = person.wca_id
     expect(merge_people).to be_invalid
-    expect(merge_people.errors.messages[:person2_wca_id]).to eq ["This person has multiple subIds", "Names don't match"]
+    expect(merge_people.errors.messages[:person2_wca_id]).to include "This person has multiple subIds"
```

Comment:
I see. I guess I'd rather have fixed this by doing something like changing

```ruby
person = FactoryGirl.create(:person_with_multiple_sub_ids)
```

to

```ruby
person = FactoryGirl.create(:person_with_multiple_sub_ids, name: person.name)
```

And leaving the checks as `eq`.

That all said, this change looks fine. I prefer to err on the side of being explicit (maybe someday someone tweaks a validation in a way that causes a 3 error to show up here, and they didn't actually mean for that to happen, being explicit about what errors we expect would for him to notice that).